edxval/models.py

```diff
@@ -447,6 +447,28 @@ def filename(self):
 
         return file_name
 
+    def save_transcript(self, file_data, file_format, file_name=None):
+        """
+        Saves Transcript Content to a Video Transcript File
+
+        Arguments:
+            file_data(InMemoryUploadedFile): Transcript content.
+            file_format(unicode): Transcript file format.
+        """
+        # generate transcript file name if not already given
+        if not file_name:
+            file_name = '{uuid}.{ext}'.format(uuid=uuid4().hex, ext=file_format)
+
+        # save the transcript file
+        if file_data:
+            with closing(file_data.open()) as transcript_content:
```

Comment:
Is the open(seek) call necessary here?

Reply:
This has been there since #87 ([click here to go to exact line](https://github.com/edx/edx-val/pull/87/files#diff-7bd2e8ec7f96c5cb70570aa07f67a7d44bd9c23309e6bdb8f587385c9f0830adR463-R466))

So I was hesitant to change something that was already there.

But I changed it, as you suggested, and the tests are running fine. Also, my manual testing went fine... 🤔